--- original
+++ modified
@@ -245,6 +245,10 @@
             <artifactId>che-plugin-zend-debugger-server</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.che.plugin</groupId>
+            <artifactId>plugin-bayesian-lsp</artifactId>
+        </dependency>
+        <dependency>
             <groupId>org.everrest</groupId>
             <artifactId>everrest-core</artifactId>
         </dependency>
