--- original
+++ modified
@@ -14,6 +14,7 @@
 import com.google.inject.multibindings.Multibinder;
 import com.google.inject.name.Names;
 
+import com.redhat.bayesian.agent.BayesianAgent;
 import org.eclipse.che.api.agent.ExecAgent;
 import org.eclipse.che.api.agent.ExecAgentLauncher;
 import org.eclipse.che.api.agent.LSCSharpAgent;
@@ -122,6 +123,7 @@
         // agents
         bind(org.eclipse.che.api.agent.server.AgentRegistry.class).to(org.eclipse.che.api.agent.server.impl.AgentRegistryImpl.class);
         Multibinder<Agent> agents = Multibinder.newSetBinder(binder(), Agent.class);
+        agents.addBinding().to(BayesianAgent.class);
         agents.addBinding().to(SshAgent.class);
         agents.addBinding().to(UnisonAgent.class);
         agents.addBinding().to(ExecAgent.class);
